pkg/catalog/inbound_traffic_policies.go

```diff
@@ -111,11 +130,24 @@ func (mc *MeshCatalog) GetInboundMeshTrafficPolicy(upstreamIdentity identity.Ser
 		routeConfigPerPort[int(upstreamSvc.TargetPort)] = append(routeConfigPerPort[int(upstreamSvc.TargetPort)], inboundTrafficPolicies)
 	}
 
-	return &trafficpolicy.InboundMeshTrafficPolicy{
-		TrafficMatches:          trafficMatches,
-		ClustersConfigs:         clusterConfigs,
-		HTTPRouteConfigsPerPort: routeConfigPerPort,
+	return routeConfigPerPort
+}
+
+func (mc *MeshCatalog) getUpstreamSvcSet(upstreamServices []service.MeshService) ([]service.MeshService, mapset.Set, mapset.Set) {
```

Comment:
Actually, I'd just recommend getting rid of this method entirely, it doesn't provide much, and there's 3 return values, none of which are always used nor do they have anything to do with each other.